images/router/haproxy/conf/haproxy-config.template

```diff
@@ -85,6 +85,10 @@ listen stats :1936
     mode http
     # Health check monitoring uri.
     monitor-uri /healthz
+{{ if (and .State .ServiceUnits) }}
```

Comment:
If there are no services in the system (or that match our shard params) will this still bind?